controllers/topolvm_controller.go

```diff
@@ -322,6 +323,28 @@ func getCsiResizerContainer() *corev1.Container {
 	return csiResizer
 }
 
+func getCsiSnapshoterContainer() *corev1.Container {
+
+	args := []string{
+		fmt.Sprintf("--csi-address=%s", DefaultCSISocket),
+		"--connection-timeout=15s",
+		"--leader-election=false",
+	}
+
+	volumeMounts := []corev1.VolumeMount{
+		{Name: "socket-dir", MountPath: filepath.Dir(DefaultCSISocket)},
+	}
+
+	csiSnapshoter := &corev1.Container{
```

Comment:
Do we dont have resource limits for this one?

Reply:
@nbalacha @Yuggupta27 ?

Reply:
I added very minimal ones, I will change them if required.